packages/commonwealth/server/migrations/20240319114920-stake-transactions.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface
+      .createTable(
```

Comment:
Doesn't seem like these 2 queries are in the same transaction.

Reply:
Good catch, fixed